### PR TITLE
Auto Refresh Date for TimeEntryForm

### PR DESCRIPTION
**Pull request by ShaunSully88**
BUG: Auto refresh/Check time and date when logging intangible time (WIP Shaun)
Right now if someone has a poor internet connection and leaves the app open for a day and goes to log intangible time, the time will be logged using the date for the last time they were active in the app. I’d like the app to check/auto-refresh the time and date to make sure it is accurate when logging this intangible time.

FIX: Not sure if correct, but I added a Today variable into the handleSubmit function that is compared to the dateOfWork in the TimeEntry conditional statement. If incorrect, looking for any feedback that can help me out. Thanks!

### CODE REVIEW

File: src/components/Timer/Timer.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import './Timer.css';
 import axios from 'axios';
 import { ENDPOINTS } from '../../utils/URL';
 
+
```
Comment (RJBornToWin) [inline src/components/Timer/Timer.jsx:10]: No actual change. Can revert this.

File: src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx [hunk 2]
```diff
@@ -399,16 +400,18 @@ const TimeEntryForm = props => {
...
     if (edit) {
       timeEntry.hours = hours;
       timeEntry.minutes = minutes;
+      inputs.dateOfWork = today;
+      console.log(dateOfWork, today);
     } else {
       timeEntry.timeSpent = `${hours}:${minutes}:00`;
     }
-
+
```
Comment (RJBornToWin) [inline src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx:414]: No actual change. Can revert this.

File: src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx [hunk 3]
```diff
@@ -32,6 +32,7 @@ import { ENDPOINTS } from '../../../utils/URL';
 import hasPermission from 'utils/permissions';
 import { getTimeEntryFormData } from './selectors';
 
+
```
Comment (RJBornToWin) [inline src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx:35]: No actual change. Can revert this.

File: package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
     "date-fns": "^2.14.0",
     "diff": "^5.0.0",
     "font-awesome": "^4.7.0",
+    "gcc": "^1.0.2",
```
Comment (RJBornToWin) [inline package.json:22]: Could you please confirm if this is intentional?

File: package-lock.json [hunk 5]
```diff
@@ -8381,6 +8381,11 @@
         "globule": "^1.0.0"
       }
     },
+    "gcc": {
```
Comment (RJBornToWin) [inline package-lock.json:8384]: Is gcc inclusion intentional ?

**Review by RJBornToWin — COMMENTED** [review]
I see the variable has been included. But I don't see any conditional checks. Have you tested your app to see if the bug is fixed ? Thanks.

File: package.json (hunk 4 above)
Comment (ShaunSully88) [reply package.json:22]: No thjis was an error on my end

**Comment by RJBornToWin** [comment]
Please remove this PR as this has been handled in another PR. Thanks.

**Pull request closed without merging**